app/views/Dashboard/Country/CountryStatItem/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {
+    isDefined,
+    _cs,
+} from '@togglecorp/fujs';
+import {
+    TextOutput,
+    Tooltip,
+} from '@the-deep/deep-ui';
+
+import styles from './styles.css';
+
+interface Props{
+    label: string;
+    value: string | undefined;
+    source?: string;
+    date?: string;
+    showRegionalValue?: boolean;
+    region?: string;
+    regionalValue?: string;
+    className?: string;
+}
+
+function CountryStatItem(props: Props) {
+    const {
+        label,
+        value,
+        source,
+        date,
+        region,
+        regionalValue,
+        showRegionalValue = false,
+        className,
+    } = props;
+
+    if (!isDefined(value)) {
+        return null;
+    }
+    return (
+        <div>
```

Comment:
This div is unencessary. You can wrap it in <></>

Reply:
This is necessary!